packages/app-desktop/commands/toggleTabKeyNavigation.ts

```diff
@@ -0,0 +1,20 @@
+import { CommandRuntime, CommandDeclaration } from '@joplin/lib/services/CommandService';
+import { _ } from '@joplin/lib/locale';
+import { DesktopCommandContext } from '../services/commands/types';
+import Setting from '@joplin/lib/models/Setting';
+
+export const declaration: CommandDeclaration = {
+	name: 'toggleTabKeyNavigation',
+	label: () => _('Toggle editor tab key navigation'),
```

Comment:
Currently, this label is only shown in keyboard shortcut settings. To avoid having to localize it, an alternative might be to get the label from the setting (as is done in `MenuBar.tsx`). However, this would result in a label of `Tab moves focus`, which doesn't describe the command's toggling action.